images/windowsserver2022/scripts/setup-3.ps1

```diff
@@ -0,0 +1,78 @@
+
+Write-Host "Set Coordinated Universal Time (UTC) time for Windows" 
+Set-ItemProperty -Path HKLM:\SYSTEM\CurrentControlSet\Control\TimeZoneInformation -Name RealTimeIsUniversal -Value 1 -Type DWord -Force
```

Comment:
Similar to some of the other Q's - I see we set UTC here, and we also set it in `autounattend.xml`:

```xml
        <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
        <!-- https://docs.microsoft.com/en-us/windows-hardware/customize/desktop/unattend/microsoft-windows-shell-setup -->
            <ComputerName>packer-win2022</ComputerName>
            <TimeZone>Coordinated Universal Time</TimeZone>
        </component>
```

Do we need both?